### PR TITLE
Add exit IPs

### PR DESCRIPTION
**Pull request by SithEngineer**

### CODE REVIEW

File: doc/next/_sources/reference/put-ad-id.rst.txt [hunk 1]
```diff
@@ -165,7 +165,6 @@ status                  2017    ad status change not allowed    Changing the sta
 totalBudget             2019    insufficient budget             Budget for ad is insufficient
 dailyBudget             2020    insufficient daily budget       Daily budget for ad is insufficient
 categoryId              2023    category deleted                The category has been deleted and will not allow creation/updating of ads
-categoryId              2026    too many ads in category        Cannot put more ads in this category
```
Comment (SithEngineer) [inline doc/next/_sources/reference/put-ad-id.rst.txt:168]: Remove incorrect information from this endpoint.

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ The component schemas are organized in separate folders per logical domain, with
 
 To help you understand the OpenAPI specification, you can [interactively explore it](https://openapi-map.apihandyman.io/?version=3.0).
 Version `3.0.0` of the OpenAPI specification is currently used.
-The most popular IDEs for development have an OpenAPI plugin, which helps with validation issues and visual inspecftion of the API.
+The most popular IDEs for development have an OpenAPI plugin, which helps with validation issues and visual inspection of the API.
```
Comment (SithEngineer) [inline README.md:13]: Fix typo.

File: src/source/reference/put-ad-id.rst [hunk 3]
```diff
@@ -165,7 +165,6 @@ status                  2017    ad status change not allowed    Changing the sta
 totalBudget             2019    insufficient budget             Budget for ad is insufficient
 dailyBudget             2020    insufficient daily budget       Daily budget for ad is insufficient
 categoryId              2023    category deleted                The category has been deleted and will not allow creation/updating of ads
-categoryId              2026    too many ads in category        Cannot put more ads in this category
```
Comment (remenska) [inline src/source/reference/put-ad-id.rst:168]: can you check if the OpenAPI yaml spec has anything about this?  these references don't need to be maintained anymore

**Review by jorgegviana — DISMISSED** [review]

File: src/source/reference/put-ad-id.rst (hunk 3 above)
Comment (SithEngineer) [reply src/source/reference/put-ad-id.rst:168]: Updated it. Thanks for the heads up.

**Review by remenska — APPROVED** [review]

**Pull request merged**